src/zkevm_specs/evm/execution/returndatacopy.py

```diff
@@ -0,0 +1,47 @@
+from ..instruction import Instruction, Transition
+from ..table import CallContextFieldTag, RW, CopyDataTypeTag
+
+
+def returndatacopy(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+
+    memory_offset_word, offset_word, size_word = (
+        instruction.stack_pop(),
+        instruction.stack_pop(),
+        instruction.stack_pop(),
+    )
+
+    return_data_length = instruction.call_context_lookup(
+        CallContextFieldTag.LastCalleeReturnDataLength, RW.Read
+    )
+    return_data_offset = instruction.call_context_lookup(
+        CallContextFieldTag.LastCalleeReturnDataOffset, RW.Read
+    )
+
```

Comment:
We need to make sure `offset_word + size_word <= return_data_length` here, Copy circuit doesn't know where is the end of return data.

Reply:
That one is missed, circuit did that but spec doesn't,  will fix soon.